HBT.cpp

```diff
@@ -106,6 +107,7 @@ int main(int argc, char **argv)
     subsnap.UpdateTracks(world, halosnap);
 
     timer.Tick(world.Communicator);
+    subsnap.WrapPositions();
```

Comment:
I think we should move the wrapping step to the last loop contained in `SubhaloSnapshot_t::UpdateTracks`. This is in an OMP loop and will reduce the addition of new function calls in the main branch. I am happy to do it myself.